core/Db/Adapter/Pdo/Mysql.php

```diff
@@ -34,6 +34,23 @@ public function __construct($config)
         if (defined('PDO::MYSQL_ATTR_LOCAL_INFILE')) {
             $config['driver_options'][PDO::MYSQL_ATTR_LOCAL_INFILE] = true;
         }
+        if ($config['enable_ssl']) {
```

Comment:
Is this change needed now that you've modified `core/Db/Adapter/Mysqli.php` maybe these file changes could be removed?